internal/usecase/repository/domain_repository.go

```diff
@@ -407,50 +420,68 @@ func (r *domainRepository) createIpaDomain(
 	return nil
 }
 
-func (r *domainRepository) updateIpaDomain(
-	db *gorm.DB,
-	data *model.Ipa,
-) (err error) {
-	if db == nil {
-		return internal_errors.NilArgError("db")
-	}
-	if data == nil {
-		return internal_errors.NilArgError("data' of type '*model.Ipa")
-	}
-	if err = db.Unscoped().
-		Delete(data).Error; err != nil {
-		return err
+func (r *domainRepository) ipaFillUpdateAgentReminders(oldData *model.Domain, newData *model.Domain) (remCerts []uint, remServers []uint, remLocations []uint) {
```

Comment:
Do you mean `remainder` (rest, residual) instead of `reminder` (recall, memory)?

Reply:
yeah! thank you! I mean `remainder`  :+1: